opentelemetry-otlp/src/proto.rs

```diff
@@ -1,4 +1,4 @@
-#[cfg(feature = "tonic")]
+#[cfg(any(feature = "tonic", feature="http-proto"))]
```

Comment:
It's good to reuse the existing proto-generated files. But this also makes `http-proto` to be dependent on the `tonic`. It may need `prost` but I don't think it need `tonic`

Reply:
not sure if there is a dependence on tonic, I'm using "any" for conditional compilation to make it true if any of the predicate is used.

Reply:
I was more referring to the fact that we use `tonic_build` here, which is not listed as a dependency for feature `http-proto`. Is it your intention to ask users to specify `tonic_build` explicitly?